betterlockscreen

```diff
@@ -6,6 +6,8 @@
 
 # find your resolution so images can be resized to match your screen resolution
 res=$(xdpyinfo | grep dimensions | sed -r 's/^[^0-9]*([0-9]+x[0-9]+).*$/\1/')
+locktext='Type password to unlock...'
```

Comment:
this line isnt required here, it is replaced by line 35 after recent change

Reply:
Oops, that must be a typo. Fixed.